### ISSUE TITLE
Claims can crash the server when they are in a biome that got removed or changed in 1.18.1.

### ISSUE THREAD

**Issue by MCTGE** [GriefPrevention/GriefPrevention#1729]
### Observed Behavior

My server crashes when Grief Prevention is loading in (seems like only the first time?) a claim that is in a biome that got removed or changes in 1.18.1.

### Expected Behavior

GP Should be updating this data without crashes.

### Reproduction steps

1. Updated a world that has claims to 1.18.1.
2. Interact with a claim that is in a changed biome.

### Stack trace or error log

```shell
[14:45:08] [Server thread/INFO]: [GriefPrevention] All of .Ghasty06's claims have expired.
[14:45:08] [Server thread/WARN]: [GriefPrevention] Task #413319 for GriefPrevention v16.17.1-2b69191 generated an exception
java.lang.NoSuchFieldError: DESERT_HILLS
at me.ryanhamshire.GriefPrevention.RestoreNatureProcessingTask.getPlayerBlocks(RestoreNatureProcessingTask.java:835) ~[GriefPrevention.jar:?]
at me.ryanhamshire.GriefPrevention.RestoreNatureProcessingTask.<init>(RestoreNatureProcessingTask.java:92) ~[GriefPrevention.jar:?]
at me.ryanhamshire.GriefPrevention.GriefPrevention.restoreChunk(GriefPrevention.java:3537) ~[GriefPrevention.jar:?]
at me.ryanhamshire.GriefPrevention.GriefPrevention.restoreClaim(GriefPrevention.java:3507) ~[GriefPrevention.jar:?]
at me.ryanhamshire.GriefPrevention.CleanupUnusedClaimTask.run(CleanupUnusedClaimTask.java:102) ~[GriefPrevention.jar:?]
at org.bukkit.craftbukkit.v1_18_R1.scheduler.CraftTask.run(CraftTask.java:101) ~[purpur-1.18.1.jar:git-Purpur-1449]
at org.bukkit.craftbukkit.v1_18_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483) ~[purpur-1.18.1.jar:git-Purpur-1449]
at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1571) ~[purpur-1.18.1.jar:git-Purpur-1449]
at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:516) ~[purpur-1.18.1.jar:git-Purpur-1449]
at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1494) ~[purpur-1.18.1.jar:git-Purpur-1449]
at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1282) ~[purpur-1.18.1.jar:git-Purpur-1449]
at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[purpur-1.18.1.jar:git-Purpur-1449]
at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

### Server version

```shell
Server: Purpur 1.18.1-R0.1-SNAPSHOT
```

### GriefPrevention version

```shell
16.17.1
```

### Configuration

```yaml
# Default values are perfect for most servers.  If you want to customize and have a question, look for the answer here first: http://dev.bukkit.org/bukkit-plugins/grief-prevention/pages/setup-and-configuration/
GriefPrevention:
  SeaLevelOverrides:
    The Golden Empire: -1
    The Golden Empire_nether: -1
    The Golden Empire_the_end: -1
  Claims:
    Mode:
      The Golden Empire_the_end: Survival
      The Golden Empire_nether: Survival
      The Golden Empire: Survival
    PreventGlobalMonsterEggs: true
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: true
    LockTrapDoors: true
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    RaidTriggersRequireBuildTrust: true
    ProtectHorses: true
    ProtectDonkeys: true
    ProtectLlamas: true
    InitialBlocks: 200
    Claim Blocks Accrued Per Hour:
      Default: 150
    Max Accrued Claim Blocks:
      Default: 100000
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: 8
    AutomaticNewPlayerClaimsRadiusMinimum: 5
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 10
    MaximumDepth: 0
    InvestigationTool: STICK
    ModificationTool: GOLDEN_SHOVEL
    Expiration:
      ChestClaimDays: 7
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 30
        ExceptWhenOwnerHasTotalClaimBlocks: 60000
        ExceptWhenOwnerHasBonusClaimBlocks: 60000
      AutomaticNatureRestoration:
        SurvivalWorlds: true
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 8
    CreationRequiresWorldGuardBuildPermission: false
    VillagerTradingRequiresPermission: false
    CommandsRequiringAccessTrust: /sethome;/pw set
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
    RavagersBreakBlocks: true
    FireSpreadsInClaims: false
    FireDamagesInClaims: false
    LecternReadingRequiresAccessTrust: false
  Spam:
    Enabled: false
    LoginCooldownSeconds: 0
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Please reduce your noise level.  Spammers will be banned.
    BanOffenders: false
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      The Golden Empire: true
      The Golden Empire_nether: true
      The Golden Empire_the_end: true
    ProtectFreshSpawns: false
    PunishLogout: false
    CombatTimeoutSeconds: 15
    AllowCombatItemDrop: true
    BlockedSlashCommands: /home;/spawn;/tpa;/pw;/tpr
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: false
      AdministrativeClaims: false
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers:
      PvPWorlds: false
      NonPvPWorlds: false
    AllowFlintAndSteelNearOtherPlayers:
      PvPWorlds: false
      NonPvPWorlds: false
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksMaxBonus: 100000
    ClaimBlocksPurchaseCost: 100.0
    ClaimBlocksSellValue: 20.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: false
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: false
  BlockSurfaceOtherExplosions: false
  LimitSkyTrees: true
  LimitTreeGrowth: false
  PistonMovement: EVERYWHERE
  PistonExplosionSound: true
  FireSpreads: true
  FireDestroys: true
  AdminsGetWhispers: true
  AdminsGetSignNotifications: true
  SmartBan: false
  Mute New Players Using Banned Words: true
  MaxPlayersPerIpAddress: 3
  Siege:
    Worlds: []
    BreakableBlocks:
    - GRASS_BLOCK
    - DIRT
    - COBBLESTONE
    - OAK_PLANKS
    - SPRUCE_PLANKS
    - BIRCH_PLANKS
    - JUNGLE_PLANKS
    - ACACIA_PLANKS
    - DARK_OAK_PLANKS
    - SAND
    - GRAVEL
    - GLASS
    - GRASS
    - FERN
    - DEAD_BUSH
    - WHITE_WOOL
    - ORANGE_WOOL
    - MAGENTA_WOOL
    - LIGHT_BLUE_WOOL
    - YELLOW_WOOL
    - LIME_WOOL
    - PINK_WOOL
    - GRAY_WOOL
    - LIGHT_GRAY_WOOL
    - CYAN_WOOL
    - PURPLE_WOOL
    - BLUE_WOOL
    - BROWN_WOOL
    - GREEN_WOOL
    - RED_WOOL
    - BLACK_WOOL
    - SNOW
    - GLASS_PANE
    DoorsOpenDelayInSeconds: 300
    CooldownEndInMinutes: 60
  EndermenMoveBlocks: false
  SilverfishBreakBlocks: false
  CreaturesTrampleCrops: false
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: true
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: true
  BanCommandPattern: ban %name% %reason%
  Advanced:
    fixNegativeClaimblockAmounts: true
    ClaimExpirationCheckRate: 60
    OfflinePlayer_cache_days: 90
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false
```

### Plugin list

```shell
Plugins (56): AdvancedAchievements, AdvancedCrates, ArmorStandTools, AuctionHouse, BisectHosting, BuycraftX, ChatItemDisplay, ChestShop, ChristmasPlus, Citizens, CitizensGUI, CMI, CMIEInjector*, CMILib, ColoredAnvils, CoreProtect, DailyRewardsPlus, DeluxeMenus, DiscordSRV, dynmap, EpicRename, ExecutableItems, FastAsyncWorldEdit (WorldEdit), GriefPrevention, Guilds, HeadDatabase, Images, ItemMagnet, Jobs, LibsDisguises, LuckPerms, mcMMO, Minecord*, PlaceholderAPI, Plan, PlayerWarps, ProtocolLib, PvPManager, Quests, SCore, ShopGUIPlus, spark, SSX-Connector*, StaffPlus, SurvivalInvisiframes, TAB, TradeMe, UltraCosmetics, Vault, ViaBackwards, ViaRewind, ViaVersion, Votifier, VotingPlugin, WorldGuard, YamipaPlugin
```

### Running without GriefPrevention

- [ ] I attempted running the server without GriefPrevention installed.
- [X] The problem does _not_ occur when GriefPrevention is removed from the server.

### Running with only GriefPrevention

- [ ] I attempted running only GriefPrevention on the server.
- [ ] The issue still occurs when GriefPrevention is the only plugin running.

### Running on a fresh, clean server installation

- [ ] I attempted testing for the issue on a new server.
- [ ] The issue still occurs on a new server.

### Using unmodified client

- [X] I attempted testing for the issue with the vanilla client.
- [X] The issue still occurs when using the vanilla client.

### We appreciate you taking the time to fill out a bug report!

- [X] I searched for similar issues before submitting this bug report.

**Comment by Revetzz** [comment]
I was about to make one as well when I saw this. I am having the same issue. Players are unable to abandonclaims, I can't delete them as an admin. My server doesn't crash but the claims are extremely buggy.

**Comment by MCTGE** [comment]
Ah, good catch on that I didn't realize the claims were not getting abandoned.

**Comment by Jikoo** [comment]
Your stack trace does not indicate a crash. This issue would also cause GP to do less processing, not more, unfortunately. It also has been fixed for a while - update to 16.18-RC1 available in the releases section here.

**Comment by Jikoo** [comment]
Duplicate of #1689
